dist/t/spec/support/capybara.rb

```diff
@@ -12,3 +13,14 @@
 
 Capybara.default_driver = :poltergeist
 Capybara.javascript_driver = :poltergeist
+
+begin
+	hostname = Socket.gethostbyname(Socket.gethostname).first
+rescue
```

Comment:
Please don't do rescue any exception or error, but only those you expect to appear.